api/views/blocks/detail.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * BlockDetailView : supports the block detail page
+ *
+ * Changes:
+ *   - 20190912.01 : initial version
+ */
+const __VERSION__ = 'BlockDetailView@1.0.0';
+
+export class BlockDetailView {
```

Comment:
the view classes enable handling so that different versions of the Network Explorer webapp can handle different versions of the API gracefully